<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Like</class>
 <widget class="QWidget" name="Form_Like">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Like</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icon/Bokehlicia-Captiva-Rocket.ico</normaloff>../Icon/Bokehlicia-Captiva-Rocket.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Like from hashtag</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Like followers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Like following</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Like last media likers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Like your timeline</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_username">
       <property name="text">
        <string>Hashtag</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_hashtag">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>love</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pb_add</tabstop>
  <tabstop>pb_apply</tabstop>
  <tabstop>pb_delete</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>le_hashtag</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
